### PR TITLE
FOIA22-168: Design Updates

### PR DESCRIPTION
**Pull request by SirJohn96**

### CODE REVIEW

File: config/default/filter.format.rich_text_wizard.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ filters:
     status: true
     weight: -50
     settings:
-      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <ul type> <ol start type> <li> <dl> <dt> <dd> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <s> <sup> <sub> <img src alt data-entity-type data-entity-uuid data-align data-caption> <table> <caption> <tbody> <thead> <tfoot> <th> <td> <tr> <hr> <p> <br> <h1> <pre> <a href hreflang class> <drupal-entity data-* data-entity-type data-entity-uuid data-entity-embed-display data-entity-embed-display-settings data-align data-caption data-embed-button data-langcode alt title>'
+      allowed_html: '<br> <p class> <h2> <h3> <h4> <h5> <h6> <a class="square" href> <span class> <h1> <div class> <strong> <em> <s> <sub> <sup> <blockquote> <ul> <ol reversed start> <li> <hr> <table> <tr> <td rowspan colspan> <th rowspan colspan> <thead> <tbody> <tfoot> <caption> <drupal-entity alt title data-align data-caption data-entity-embed-display data-entity-embed-display-settings data-view-mode data-entity-uuid data-langcode data-embed-button="media_browser node" data-entity-type="media node">'
```
Comment (mrclay) [inline config/default/filter.format.rich_text_wizard.yml]: The reordering here makes it difficult to review this change. Can you undo this change and just add the new things at the end?

File: config/default/editor.editor.rich_text_wizard.yml [hunk 2]
```diff
@@ -38,7 +38,11 @@ settings:
       reversed: true
       startIndex: true
     ckeditor5_sourceEditing:
-      allowed_tags: {  }
+      allowed_tags:
+        - '<p class>'
+        - '<span class>'
+        - '<h1>'
+        - '<div class>'
```
Comment (mrclay) [inline config/default/editor.editor.rich_text_wizard.yml:45]: @mattdanger This will only affect the wizard's HTML editors, right? Seems OK to me.

Comment (mattdanger) [reply config/default/editor.editor.rich_text_wizard.yml:45]: Correct, this text format is only used by the Wizard. So this should be fine 👍

File: config/default/filter.format.rich_text_wizard.yml (hunk 1 above)
Comment (SirJohn96) [reply config/default/filter.format.rich_text_wizard.yml]: This has been addressed.
